baybe/recommenders/pure/bayesian/botorch.py

```diff
@@ -47,14 +48,24 @@ class BotorchRecommender(BayesianRecommender):
     # See base class.
 
     # Object variables
+    num_restarts: int = field(validator=[instance_of(int), gt(0)], default=10)
+    """Controls how many times gradient-based optimization is restarted from different
+    initial points. **Does not affect purely discrete optimization**.
+    """
+
+    raw_samples: int = field(validator=[instance_of(int), gt(0)], default=64)
+    """Controls the number of raw samples drawn for the initialization heuristic in
+    gradient-based optimization. **Does not affect purely discrete optimization**.
```

Comment:
```suggestion
    gradient-based optimization. **Does not affect discrete optimization.**
```

Reply:
would you really prefer that? I adde the `purely` to make it crystal clear that ONLY pure discrete optimization is not affected, so hybrid and conti are. When I read it according to the suggesitons theres always a doubt like `when they say discrete do they also mean hyrbid because hybrid has discrete optimziaiton parts, right?`, you know what I mean?

Reply:
ultimately this is ambigous, you seem to split the hybrid optimization into discrete and conti insted of regarding it as its own type -  even in that scenario the discrete optimization is a purely discrete optimization. However, our search space type explicitly has HYBRID as a type so I'll keep the original